### DISCUSSION TITLE
Misusing publish_image --> Seeking Alternatives

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#227] [Q&A] [answered]
In an attempt to get the 'last detected image', I tried using `publish_image `.  It works but it is noted it shouldn't be used in the way I'm using it:

`Note: this will use some extra CPU and should probably only be used for debugging`

Is there a better way to get the last detected image for a camera **other than using thumbnail**? I'm using the thumbnail configuration as a way to preface what was recorded (I look through the thumbnails to determine which video to watch). Hence `publish_image ` was working.  Ideally I'm trying to send that last detected image via a notification. (Am I going about this wrong?)

**Comment by roflcoopter** [comment, marked as answer]
There are two ways to go about this
- Under `recorder` you can enable `send_to_mqtt` which will create a camera entity in Home Assistant called `camera.{client_id from mqtt config}_{mqtt_name from camera config}_latest_thumbnail`
- Since you are already saving the thumbnail to disk, you could grab that file from Home Assistant and attach it in a notification

**Reply by gniknalu** [reply]
Thank you - I'll use `send_to_mqtt` 

The images are one one Linux machine and my Home Assistant installation is on another so grabbing the file - although entirely possible - will add complexity. So I'll go with your first suggestion. Thank you!
